lib/tasks/check_content_consistency.rake

```diff
@@ -1,11 +1,16 @@
 namespace :check_content_consistency do
-  def check_content(base_path)
-    checker = ContentConsistencyChecker.new(base_path)
+  def check_content(base_path, ignore_recent = false)
+    checker = ContentConsistencyChecker.new(base_path, ignore_recent)
     errors = checker.call
 
     if errors.any?
-      puts "#{base_path} 😱"
-      puts errors
+      Airbrake.notify(
+        "Found an inconsistent content item: #{base_path} 😱",
```

Comment:
Same as https://github.com/alphagov/publishing-api/pull/852#discussion_r107231488. This might overload Errbit's email system.